viz.yaml

```diff
@@ -990,6 +1008,20 @@ publish:
     context:
       projects: projects
   -
+    id: flipClock
+    publisher: section
+    scripts: scripts/publish/
+    template: layout/templates/flipClock.mustache
+    depends:
+      flipClockFlips: flipClockFlips
+    context:
+      flipClockFlips: flipClockFlips
+  -
+    id: flipClockFlips
```

Comment:
I don't remember how the partials will work, I'll have to play with this to figure it out.